### PR TITLE
feat: Cluster-scope maintenance guide

### PR DESCRIPTION
**Pull request by tumido**
Resolves: https://github.com/operate-first/apps/issues/164

Related: https://github.com/operate-first/apps/pull/180 https://github.com/operate-first/apps/pull/178

### CODE REVIEW

**Comment by tumido** [comment]
Btw, there might be another exception to this that might need mentioning and that's crds, since there may exist CRDs with the same name in different API groups so this structure may be wrong for CRDs. WDYT?

**Comment by tumido** [comment]
I mean, the motivation is to enforce consistency... I'd really like us to crave for perfection in this case, because we need to gain trust that this way can be used to effectively manage clusters.

**Comment by tumido** [comment]
Also: linking relevant discussion from https://github.com/operate-first/apps/pull/178#issuecomment-769793299

File: cluster-scope/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Cluster management application organization
+
+This is just a short addendum to [ADR 0009](https://www.operate-first.cloud/blueprints/blueprint/docs/adr/0009-cluster-resources.md) declaring the direct consequences and organization structure of this `cluster-scope` application. This are the best practises we expect every maintainer who does changes to this application to follow.
```
Comment (HumairAK) [inline cluster-scope/README.md:3]: > This are the best 

to

>These are the best

**Comment by HumairAK** [comment]
> Btw, there might be another exception to this that might need mentioning and that's crds, since there may exist CRDs with the same name in different API groups so this structure may be wrong for CRDs. WDYT? 

This seems like a pretty niche case, I'm good with just doing something like: 

`cluster-scope/base/crds/prow.k8s.io/apigroupA/kustomization.yaml`
and:
`cluster-scope/base/crds/prow.k8s.io/apigroupB/kustomization.yaml` 
So we can stay somewhat consistent.

**Comment by HumairAK** [comment]
Looks great, I would imagine in the future we can place CI guards against potential changes to downstream environments. 

/lgtm

**Pull request merged**